projects/js-packages/publicize-components/src/components/form/index.js

```diff
@@ -70,6 +75,43 @@ export default function PublicizeForm( {
 		[ autosave, isEditedPostDirty ]
 	);
 
+	const renderNotices = () => (
+		<>
+			{ brokenConnections.length > 0 && (
+				<Notice type={ 'error' }>
+					{ createInterpolateElement(
+						_n(
+							'One of your social connections is broken. Reconnect them on the <fixLink>connection management</fixLink> page.',
+							'Some of your social connections are broken. Reconnect them on the <fixLink>connection management</fixLink> page.',
+							brokenConnections.length,
+							'jetpack'
+						),
+						{
+							fixLink: <ExternalLink href={ connectionsAdminUrl } />,
+						}
+					) }
+				</Notice>
+			) }
+			{ unsupportedConnections.length > 0 && (
+				<Notice type={ 'error' }>
+					{ createInterpolateElement(
+						_n(
+							'One of your social connection is not supported anymore. <moreInfo>Learn more here</moreInfo>.',
```

Comment:
I think 'connection' should still be plural here

```suggestion
							'One of your social connections is not supported anymore. <moreInfo>Learn more here</moreInfo>.',
```

I was wondering if we should change this to 'Twitter is not supported anymore', and then we don't have to worry about how many unsupported connections there are.